src/features/query-bar/components/filter/Condition.js

```diff
@@ -159,7 +160,7 @@ const Condition = ({
     const renderDeleteButton = () => {
         return (
             <div className="filter-item-delete-button">
-                <button type="button" className="delete-button" onClick={onDeleteButtonClick}>
+                <button className="delete-button" onClick={onDeleteButtonClick}>
```

Comment:
Please make sure to always provide a `type` for button elements, as they are `type="submit"` by default.